addons/udes_security/tests/test_desktop_access.py

```diff
@@ -0,0 +1,201 @@
+from contextlib import contextmanager
+
+from odoo.tests import common
+from odoo.tools import mute_logger, config
+from odoo import http, api
+
+
+class DesktopAccessBase(common.HttpCase):
+    def setUp(self):
+        """
+        Use test cursor for environment
+        Create two users for testing and set their passwords so logging in
+        from controllers can be simulated
+        """
+        super().setUp()
+
+        # Use default test cursor for default environment
+        def restore(cr=self.cr, env=self.env):
```

Comment:
Is it worth making a new class for both `DesktopAccessBase` and `TestFileControllers` to inherit?

As this will likely be needed if we need to test controllers elsewhere for security fun.

Maybe `ControllerBase` that has the setup simply include this method?

Reply:
And the `release` method I guess.

Happy for you to make the call though. :)

Reply:
Yeah you're right, i'll try it out